flake.nix

```diff
@@ -40,10 +35,17 @@
         ]);
       }));
 
+      openssl-static = pkgsUnstable.openssl.override { static = true; };
+
     in
     {
       packages.mkosi = mkosiDev;
 
+      packages.openssl = pkgsUnstable.symlinkJoin {
```

Comment:
What is happening here?

Reply:
`symlinkJoin` is an efficient method to combine the output of two derivations. This way, we get the headers and library (`.a`) in one derivation that can be used by `rules_nixpkgs` / Bazel cc_library rules.

This idea is also used here: https://github.com/tweag/rules_nixpkgs/blob/7e627d76ba65d6c42586fc265e46bd370672b4eb/examples/toolchains/rust/openssl-static.nix